vue/components/ui/organisms/details-expandable/details-expandable.stories.js

```diff
@@ -0,0 +1,64 @@
+import { storiesOf } from "@storybook/vue";
+import { withKnobs } from "@storybook/addon-knobs";
+
+storiesOf("Components/Organisms/Details Expandable", module)
+    .addDecorator(withKnobs)
+    .add("Details Expandable", () => ({
```

Comment:
Some component props are missing from the story I think ?

Reply:
added the rest